kafka-client/src/main/java/dev/responsive/internal/stores/CommitBuffer.java

```diff
@@ -385,6 +390,11 @@ private RemoteWriteResult flush(final long offset, final int batchSize) {
     );
     buffer.clear();
 
+    if (offset - lastOffsetLogged > INFO_LOGGING_OFFSET_INTERVAL) {
```

Comment:
if we incorporate my comment above then we can just remove this and the offset interval altogether, right? (and just log on every flush, which we seem to be doing anyway - yes that'll be a lot when there's 4096 virtual partitions, but I did find that to be really helpful debugging stuff)

Reply:
Fair enough, we can always reduce it further or implement the interval/summary logging if it's really overwhelming. Let's keep an eye on it for now. Maybe I'm just being paranoid after one too many on-call escalations that came down to performance degradation from excessive logging

(the other one I'm a bit concerned about is the INFO-level logging after each successful changelog truncation, though I left that alone for now as well -- of course logging should be trivial compared to the fairly heavyweight admin  requests to delete records, so...